liblouis/compileTranslationTable.c

```diff
@@ -3820,11 +3821,15 @@ compileRule (FileInfo * nested)
 		case CTO_Match:
 		{
 			CharsString ptn_before, ptn_after;
-			widechar patterns[27720];
 			TranslationTableOffset offset;
 			int len, mrk;
 
-				memset(patterns, 0xffff, sizeof(patterns));
+			size_t patternsByteSize = sizeof(*patterns) * 27720;
+			patterns = (widechar*) malloc(patternsByteSize);
+			if(!patterns)
+				outOfMemory();
+			memset(patterns, 0xffff, patternsByteSize);
```

Comment:
The initialization to `0xffff` was already being done before. To be honest I was not sure if removing that could cause any regressions, so I preferred to keep it as close as possible to what we had and just focus on dynamically allocating that variable and initializing it in an equivalent way.